src/main/java/com/powsybl/openloadflow/network/impl/LfAggregatedLoadsImpl.java

```diff
@@ -112,6 +114,26 @@ public double getLoadTargetQ(double diffLoadTargetP) {
         return newLoadTargetQ;
     }
 
+    @Override
+    public boolean isDisabled(String originalId) {
+        return loadsStatus.get(originalId);
+    }
+
+    @Override
+    public void setDisabled(String originalId, boolean disabled) {
+        loadsStatus.put(originalId, disabled);
+    }
+
+    @Override
+    public Map<String, Boolean> getLoadsStatus() {
+        return loadsStatus;
+    }
+
+    @Override
+    public void setLoadsStatus(Map<String, Boolean> loadsStatus) {
+        this.loadsStatus = loadsStatus;
+    }
```

Comment:
my understanding is that these get/set methods are used only for the purpose of saving and restoring a BusDcState. In order to avoid the map to be manipulated outside this class in the future, maybe we should:
- for the getter: return an immutable copy of the map
- for the setter: make a copy of the input map

?